202009.0/ffa6653a-7d0e-41cc-a24e-074752b2ce1e.md

```diff
@@ -1,14 +1,10 @@
-> Audience:
-> - Everyone who is not familiar with the Docker SDK.
->
-> Outcome:
-> - You know how to quickly set up Spryker in a local environment.
+In this guide we are going to launch an existing Spryker project in Docker containers on a local developemnt machine,
```

Comment:
Clearly set epxectations.

Reply:
Focus only on Dev mode. Once concern per page, please.
Production specification is more complex than a snippet on this page :)